keps/prod-readiness/sig-node/3085.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3085
+alpha:
+  approver: "@ehashman"
```

Comment:
you may want to change it to somebody who is available for 1.25 review

Reply:
assigned to John based on chat in slack.